main.tf

```diff
@@ -91,7 +92,7 @@ resource "aws_security_group" "bastion" {
     from_port       = "22"
     to_port         = "22"
     protocol        = "tcp"
-    security_groups = ["${module.aws_elb.security_group_id}"]
+    security_groups = ["${module.aws_elb.security_group_ids}"]
```

Comment:
See comment for version change above. `0.1.8` only returns `security_group_id`, but not id**s**

Reply:
Nice catch 🦅 👁️ 
But not enough. I used a moving tag instead:
Before:
<img width="687" alt="Screenshot 2020-12-17 at 11 26 02" src="https://user-images.githubusercontent.com/127131/102476205-1e54af00-405b-11eb-8e36-ddb8bb765552.png">

After:
<img width="900" alt="Screenshot 2020-12-17 at 11 28 06" src="https://user-images.githubusercontent.com/127131/102476221-23196300-405b-11eb-9cb6-36dcdbb210b6.png">

So the release tag can actually stay as-is.